roles/reproducer/tasks/configure_computes.yml

```diff
@@ -18,3 +18,28 @@
       ansible.builtin.command:
         cmd: |
           ping -c2 {{ _controller_ip4 }}
+
+    - name: RHEL related tasks on computes
+      become: true
+      when:
+        - cifmw_repo_setup_rhos_release_rpm is defined
+      block:
+        - name: Get rhos-release and CA
+          ansible.builtin.import_tasks: rhos_release.yml
+
+        - name: Configure rhos-release
+          ansible.builtin.command:
+            cmd: "rhos-release {{ cifmw_repo_setup_rhos_release_args }}"
+
+    - name: Create repositories on computes
```

Comment:
compute nodes?

Reply:
`computes` matches the group name in the ansible inventory. Both are valid imho.

Reply:
oh, that makes sense. :+1: